.github/workflows/auto-publish-docker-dev-branch.yml

```
@@ -0,0 +1,36 @@
+name: Upload Docker Image of Latest Developer Branch to GHCR
+
+on:
+  schedule:
+    - cron: "0 16 * * *"  # Daily at noon EST
+  workflow_dispatch:
+  push:
+
+concurrency:  # Cancel previous workflows on the same pull request
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  release-image:
+    name: Release NeuroConv Image of Current Dev Branch
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ secrets.DOCKER_UPLOADER_USERNAME }}
+          password: ${{ secrets.DOCKER_UPLOADER_PASSWORD }}
+      - name: Build and push
+        uses: docker/build-push-action@v3
+        with:
+          push: true  # Push is a shorthand for --output=type=registry
+          tags: ghcr.io/catalystneuro/neuroconv:dev
```

```suggestion
          tags: ghcr.io/catalystneuro/neuroconv:main
```

One question is how we want the current `main` branch to be tagged - should this be `dev`, `main`, or something else?

@weiglszonja @bendichter @luiztauffer

---

`latest` is another standard keyword

---

@CodyCBakerPhD it might be useful to have one at least two: one for dev and one for main?
```
tags: ghcr.io/catalystneuro/neuroconv:${{ github.ref_name }}
```

although I agree that, when built from main, the standard tag should be `latest`

---

latest releases should have two tags now: `latest` and the exact version

New releases should swap the `latest` tag to the newer one automatically but the exact version should still stick around forever

Current `main` branch is built daily or on manual dispatch with tag `dev`